hyperion-timber/src/main/java/com/willowtreeapps/hyperion/timber/list/TimberLogListActivity.java

```diff
@@ -54,4 +61,18 @@ public boolean onSupportNavigateUp() {
         throw new RuntimeException("TimberLogTree not planted in forest.");
     }
 
-}
\ No newline at end of file
+    @Override
+    public void onClick(View view) {
+        int size = getLogItemQueue().size();
+        StringBuilder logs = new StringBuilder();
+        for(int i = 0; i < size; i++) {
+            logs.append(getLogItemQueue().getItem(i).level).append(" : [").append(getLogItemQueue().getItem(i).date).append("] --> ").append(getLogItemQueue().getItem(i).message +" \n");
```

Comment:
This is really difficult to read. Can you clean this up a bit?

```java
CircularBuffer<LogItem> logItemQueue = getLogItemQueue();
for(int i = 0; i < size; i++) {
   LogItem logItem = logItemQueue.getItem(i);
   logs.append(logItem.level + " : [" + logItem.date + "] --> " + logItem.message + "\n");
}
```

Note the multiple append calls are actually somewhat expensive, especially so when they are being called hundreds of times on the main thread.

Reply:
<img width="1569" alt="screen shot 2018-12-03 at 6 19 37 pm" src="https://user-images.githubusercontent.com/16566323/49414432-3b461000-f728-11e8-8628-4bc5ad3f0c35.png">